content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -222,7 +222,42 @@ Below are the procedures to set up autocompletion for PowerShell.
    ```powershell
    del kubectl-convert.exe kubectl-convert.exe.sha256
    ```
+### Uninstall kubectl on Windows
 
+Depending on how you installed `kubectl`, use one of the following methods.
+
+ {{< tabs name="kubectl_win_uninstall" >}}
+
+   {{% tab name="curl" %}}
+   Locate the installation folder of `kubectl` on your Windows system by navigating to the appropriate directory and use the below code.
+   ```powershell
+   del kubectl.exe kubectl.exe.sha256
+   ```
+   By default, `kubectl` is installed in the `C:\Program Files\` directory.
+   {{% /tab %}}
+
+   {{% tab name="winget" %}}
+   Uninstall package using winget:
+   ```powershell
+   winget uninstall Kubernetes.kubectl
+   ```
+   {{% /tab %}}
+
+   {{% tab name="choco" %}}
+   ```powershell
+   choco uninstall kubernetes-cli
+   ```
+   Generally, the packages installed using chocolatey are downloaded into the ```C:\ProgramData\chocolatey\lib\```.
+   {{% /tab %}}
+   
+   {{% tab name="scoop" %}}
+   ```powershell
+   scoop uninstall kubectl
+   ```
+   {{% /tab %}}
+
+{{< /tabs >}}
+   
```

Comment:
How about adding the 
> After the removal process completes, you can verify that kubectl is no longer installed by running:

text here?

Reply:
Please respond to the comment @SpringWiz11